src/redfishpower/redfishpower.c

```diff
@@ -1148,6 +1150,10 @@ static void auth(char **av)
         printf("Usage: auth user:passwd\n");
         return;
     }
+    if (userpwd_set_on_cmdline) {
+        printf("auth set on cmdline\n");
```

Comment:
What is the purpose of the output here?

Reply:
just informative?  I guess it's not super necessary.